clients/cobol-lsp-vscode-extension/src/services/ProcessorGroups.ts

```diff
@@ -25,32 +25,48 @@ import {
   normalizePath,
 } from "./util/FSUtils";
 import { SettingsService } from "./Settings";
+import {
+  B4GTypeMetadata,
+  decodeBridgeJson,
+  loadBridgeJsonContent,
+} from "./BridgeForGit";
 
 const PROCESSOR_GROUP_FOLDER = ".cobolplugin";
 const PROCESSOR_GROUP_PGM = "pgm_conf.json";
 const PROCESSOR_GROUP_PROC = "proc_grps.json";
 
+type ProcessorGroupConfiguration = {
+  // Maps path or glob to processor group name
+  groupMap: { [key: string]: string };
+  procGroups: ProcessorConfig[];
+};
+
 export function loadProcessorGroupCopybookPaths(
   documentUri: string,
   dialectType: string,
 ): string[] {
-  return loadProcessorGroupSettings(
-    documentUri,
-    "libs",
-    [] as string[],
-    dialectType,
-  );
+  const pgCfg: ProcessorConfig | undefined =
+    loadProcessorsConfigForDocument(documentUri);
+  if (pgCfg === undefined) {
+    return [];
+  }
+  return loadProcessorGroupSettings(pgCfg, "libs", [] as string[], dialectType);
```

Comment:
This sequence (loadProcessorsConfigForDocument, check undefined, call loadProcessorGroupSettings) is repeated all over the place.
It looks like returning the third parameter as the default answer on `loadProcessorsConfigForDocument` failure keeps being the expected behavior.